docs/gateway/gcs-limitations.md

```diff
@@ -0,0 +1,10 @@
+## Minio GCS Gateway Limitations
+
+Gateway inherits the following GCS limitations:
+
+- Maximum number of multiparts is 1024.
+- No support for prefix based bucket policies. Only top level bucket policy is supported.
```

Comment:
Bucket policy is not supported completely. Only bucket can have read-only, write-only policies. Objects do not inherit these policies from bucket yet.